service/dynamodb/dynamodbattribute/encode.go

```diff
@@ -189,6 +189,10 @@ type MarshalOptions struct {
 	//
 	// Enabled by default.
 	SupportJSONTags bool
+
+	// Similarly supported for the YAML struct tag.
+	// Enabled by default.
```

Comment:
Couldn't be enabled by default, and would want to document which `YAML` tags are supported and what they mean.